### ISSUE TITLE
Yj -> jY

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/BOR#2]
In Sanskrit words, 'Yj' needs to be changed to 'jY' in many cases (mostly involving root jYA to know).
Refer also to https://github.com/sanskrit-lexicon/csl-orig/issues/938

**Comment by drdhaval2785** [comment]
Yj is also is a valid consonant pair. Generic change may introduce unwanted errors. piYj, aYj etc are all valid Sanskrit words.

**Comment by funderburkjim** [comment]
Agree.  That's why I didn't do a global change.  It will be necessary to examine all the several-hundred cases so that only 'correct' changes will be made.  Feel free to do the required analysis :)  Otherwise, I'll get to it sometime.

**Comment by gasyoun** [comment]
> necessary to examine all the several-hundred cases

Hope it's not high in the priority list, as BOR is a 2nd tier dictionary.
